src/sentry/apidocs/registry.py

```diff
@@ -0,0 +1,756 @@
+import sentry.api.endpoints.accept_organization_invite
```

Comment:
why not just have an `@private` decorator (or private attribute)

Reply:
I like this idea -- what do we think about:

1. in the base Endpoint class, set an attribute `public` equal to `None`
2. Our docs processor will check that the public attribute has been set to True or False on the child endpoints and give a helpful error message / link to site about our documentation efforts if the developer hasn't set it
3. Keep this exclusion list here for existing endpoints (Want to avoid having to change hundreds of files)

The purpose of this change is to cause the developer to make an explicit decision about the publicness / privateness of their endpoint, so having the `public` attribute be default `False` wouldn't be doing much for us in that regard. This is something we could enforce through review instead, but enforcing through code seems ideal.

Reply:
That works for me. Let's make sure put a comment in this file what its purpose was at time of creation and the proper way to label endpoints as private from here on out.

Also let's rename this from `registry.py` to something more descriptive like `private_endpoint_registry.py`

Reply:
@volokluev I've went ahead with having two attributes -- `public` and `private` -- the user can either set public to a list of HTTP verbs that will be public on the endpoint, or set `private` to True. and I've renamed registry.py to `src/sentry/apidocs/public_exclusion_list.py` and added a descriptive comment to the top of the file.

Reply:
well done